test/test_endpoints.py

```diff
@@ -248,3 +249,25 @@ def test_error_response_class_by_hash():
     assert resp.status_code == 500
     expected_message = f"Class with hash {INVALID_HASH} is not declared"
     assert expected_message == error_message
+
+@devnet_in_background()
+def test_create_block_endpoint():
+    """test empty block creationn"""
+    resp = get_block_by_number({"blockNumber": "latest"}).json()
+    assert resp.get("block_hash") == GENESIS_BLOCK_HASH
+    assert resp.get("block_number") == GENESIS_BLOCK_NUMBER
+
+    resp = requests.post(f"{APP_URL}/create_block").json()
+    assert resp.get("block_number") == GENESIS_BLOCK_NUMBER + 1
+    assert resp.get("block_hash") == hex(GENESIS_BLOCK_NUMBER + 1)
+    assert resp.get("status") == "ACCEPTED_ON_L2"
+    assert resp.get("gas_price") != "0"
```

Comment:
Isn't gas_price a number? So this will always be true. Perhaps it would be better to provide a gas price to `@devnet_in_background(...)` and then assert that the value in the response is equal to that.

Reply:
Interesting question, first time I noticed it I was also confused then I concluded that it is actually fine but /estimate_fee is not. 
All endpoints I tested return numbers as hex-string only /estimate_fee return int.
/get_block return gas_price as hex-string and so does /create_block.
So gas_price is a hex-string that will be  '0x174876e800' (1e11) if setting a gas fee and genesis block creation work.
So with one hit, I kill 2 flies without the need to duplicate.

But I agree it looks weird and cleaner will be to test against 1e11

Reply:
9fcf2d526c62fe1dbc5112dd250fe54bc3727fe8

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/189/commits/7d95ac2e66a71664c0c8b750982e73c7db4c854c